res/lang/bossbetty/bossbetty_en.txt

```diff
@@ -1,153 +1,153 @@
-DIALOGUE_BOSS_MAIN_TEXT_AWARDNAME_ENDS_TOMORROW1 = Don't just sit here: tomorrow the %AWARDNAME% will be awarded!
-DIALOGUE_BOSS_MAIN_TEXT_AWARDNAME_ENDS_TOMORROW2 = Do I look like the tooth fairy - what are you waiting for? Tomorrow the %AWARDNAME% will be awarded!
-DIALOGUE_BOSS_MAIN_TEXT_AWARDNAME_ENDS_TODAY1    = Up, up, step up! Today the %AWARDNAME% will be awarded!
-DIALOGUE_BOSS_MAIN_TEXT_AWARDNAME_ENDS_TODAY2    = Since you are standing around gazing stupidly: today the %AWARDNAME% will be awarded!
+DIALOGUE_BOSS_MAIN_TEXT_AWARDNAME_ENDS_TOMORROW1 = Don't just sit there: tomorrow the %AWARDNAME% will be awarded!
+DIALOGUE_BOSS_MAIN_TEXT_AWARDNAME_ENDS_TOMORROW2 = Do I look like the tooth fairy? What are you waiting for? Tomorrow the %AWARDNAME% will be awarded!
+DIALOGUE_BOSS_MAIN_TEXT_AWARDNAME_ENDS_TODAY1    = Get going! Step up! Today the %AWARDNAME% will be awarded!
+DIALOGUE_BOSS_MAIN_TEXT_AWARDNAME_ENDS_TODAY2    = As long as you're standing around, gawking: today the %AWARDNAME% will be awarded!
 DIALOGUE_BOSS_MAIN_TEXT_NEW_AWARDNAME_TOMORROW1  = Starting tomorrow, a new Sammy will be awarded. I want us to get the %AWARDNAME%!
-DIALOGUE_BOSS_MAIN_TEXT_NEW_AWARDNAME_TOMORROW2  = Put more effort into it: starting tomorrow, points are up for grabs for the new %AWARDNAME%!
-DIALOGUE_BOSS_MAIN_TEXT_PROGRAMMEMALFUNCTION1    = This incompetence is incredible. You didn't send any programme |b|%PROGRAMMEMALFUNCTION%x|/b|.
+DIALOGUE_BOSS_MAIN_TEXT_NEW_AWARDNAME_TOMORROW2  = Put more effort in: starting tomorrow, points are up for grabs for the new %AWARDNAME%!
+DIALOGUE_BOSS_MAIN_TEXT_PROGRAMMEMALFUNCTION1    = This incompetence is incredible!!! You didn't broadcast any programme |b|%PROGRAMMEMALFUNCTION%x|/b|!
 DIALOGUE_BOSS_MAIN_TEXT_NEWSMALFUNCTION1         = Instead of not broadcasting any news |b|%NEWSMALFUNCTION%x|/b|, you could at least transmit last year's weather report!
-DIALOGUE_BOSS_MAIN_ENDING_DEFAULT1               = Back to work or someone else will get your job...!
-DIALOGUE_BOSS_MAIN_ENDING_UNFRIENDLY1            = My son's golden hamster could do your job better than you!
-DIALOGUE_BOSS_MAIN_ENDING_UNFRIENDLY2            = One more blunder like that and your job will go to the next best person knocking on the door.
-DIALOGUE_BOSS_MAIN_TEXT_OTHERPLAYER1             = Get out - and pass on the worst regards to your boss. Harr harr harr...
-DIALOGUE_BOSS_MAIN_TEXT_OTHERPLAYER2             = Out! I can't stand to watch you and your damn channel any longer!
+DIALOGUE_BOSS_MAIN_ENDING_DEFAULT1               = Back to work or someone else will get your job!
+DIALOGUE_BOSS_MAIN_ENDING_UNFRIENDLY1            = My son's hamster could do your job better than you!
+DIALOGUE_BOSS_MAIN_ENDING_UNFRIENDLY2            = One more blunder and your job will go to the next person who knocks on the door!
+DIALOGUE_BOSS_MAIN_TEXT_OTHERPLAYER1             = Get out - and pass on my worst regards to your boss. Hahahahaha...
```

Comment:
Whats wrong with this "more evil" laughing style of "harr harr harr" (pirate like ? )

Reply:
Only pirates laugh like that and the other players aren't pirates.  :-)

Reply:
Actually I think I should change it to "Hahahahaaaa!" - a good compromise and it sounds a bit more evil.